src/main/kotlin/com/github/vacxe/googleplaycli/actions/Tracks.kt

```diff
@@ -10,40 +10,40 @@ import com.google.api.services.androidpublisher.model.TrackRelease
 import com.google.api.services.androidpublisher.model.TracksListResponse
 
 interface Tracks : BaseAction {
-    fun tracksList(model: DefaultModel): TracksListResponse {
+    fun tracksList(model: EditModel): TracksListResponse {
         val edits: AndroidPublisher.Edits = androidPublisher.edits()
-        val insert = edits.insert(model.packageName, null).execute()
-        return edits.tracks().list(model.packageName, insert.id).execute()
+        val editId = model.editId ?: edits.insert(model.packageName, null).execute().id
+        return edits.tracks().list(model.packageName, editId).execute()
     }
 
     fun tracksGet(model: TracksGetModel): Track {
         val edits: AndroidPublisher.Edits = androidPublisher.edits()
-        val insert = edits.insert(model.packageName, null).execute()
-        return edits.tracks().get(model.packageName, insert.id, model.track).execute()
+        val editId = model.editId ?: edits.insert(model.packageName, null).execute().id
+        return edits.tracks().get(model.packageName, editId, model.track).execute()
     }
 
     fun tracksPatch(model: TracksPatchModel): Track {
         val edits: AndroidPublisher.Edits = androidPublisher.edits()
-        val insert = edits.insert(model.packageName, null).execute()
+        val editId = model.editId ?: edits.insert(model.packageName, null).execute().id
 
         val track = Track().setReleases(
                 listOf(TrackRelease().apply {
                     versionCodes = mutableListOf(model.apkVersionCode.toLong())
                     userFraction = model.userFraction
                 })
         )
-        return edits.tracks().patch(model.packageName, insert.id, model.track, track).execute()
+        return edits.tracks().patch(model.packageName, editId, model.track, track).execute()
     }
 
     fun tracksUpdate(model: TracksUpdateModel): Track {
         val edits: AndroidPublisher.Edits = androidPublisher.edits()
-        val insert = edits.insert(model.packageName, null).execute()
 
-        val track = Track().setReleases(
-                listOf(TrackRelease().apply {
-                    versionCodes = mutableListOf(model.apkVersionCode.toLong())
-                })
-        )
-        return edits.tracks().update(model.packageName, insert.id, model.track, track).execute()
+        val release = TrackRelease()
+                .setVersionCodes(listOf(model.apkVersionCode.toLong()))
+                .setStatus("completed")
+
+        val track = Track().setReleases(listOf(release))
+
+        return edits.tracks().update(model.packageName, model.editId, model.track, track).execute()
```

Comment:
shouldn't this have the same logic as others `model.editId ?: edits.insert(model.packageName, null).execute()`? as I see `TracksUpdateModel#editId` is nullable